<?xml version="1.0"?>
<launch>
  <arg name="camera_file" default="$(find rct_examples)/data/test_set_10x10/camera_intr_calibrated.yaml"/>
  <arg name="target_file" default="$(find rct_examples)/data/test_set_10x10/target.yaml"/>
  <arg name="data_path" default="$(find rct_examples)/data/test_set_10x10/cal_data.yaml"/>

  <node pkg="rct_examples" type="intr_camera_cal_ex" name="intrinsic_cal_example" output="screen">
    <rosparam command="load" file="$(arg camera_file)"/>
    <rosparam command="load" file="$(arg target_file)"/>
    <param name="data_path" value="$(arg data_path)"/>
  </node>
</launch>
